src/Popover/hooks/useDimensions.js

```diff
@@ -50,14 +51,20 @@ const useDimensions: UseDimensions = ({ containerRef, popover, content, fixed })
         });
       }
     };
+    const scrollableParrent = getScrollableParent(containerRef.current);
```

Comment:
One small thing @filipdanisko. Because using this recursion function can be quite expensive, I would run it only the very first render with `useMemo` and empty dependencies `[]`. IMHO, it's impossible that the parent element will change without affection it's children (not triggering unmount).

Reply:
I can't really call useMemo here, that would go against Hook usage rules 🤔